draft/draft-ietf-tcpm-yang-tcp.xml

```diff
@@ -176,7 +190,7 @@
 
             <t>Connection parameters: Many implementations have means to
             influence the behavior of each TCP connection, e.g., on the
-            programming interface used by applications. A typical example are
+            programming interface used by applications. Typical examples are
```

Comment:
This is one example only. Did you mean to add more examples here, and why you make the sentence plural?

Reply:
This was a review comment from Med. But as there are many different socket options (see next line), plural somehow makes sense to me. The document currently just mentions one such socket option (TCP_NODELAY), but there are more.

I am not a native speaker. Do I miss something?